ccd-definition/CaseEventToFields.json

```diff
@@ -245,7 +245,32 @@
     "PageID": 1,
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
-    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/enter-applicant/mid-event"
+    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/enter-applicant/mid-event",
+    "FieldShowCondition": "applicantsMigrated=\"No\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "enterApplicant",
+    "CaseFieldID": "applicants",
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "OPTIONAL",
+    "PageID": 1,
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "applicantsMigrated=\"Yes\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "enterApplicant",
+    "CaseFieldID": "applicantsMigrated",
+    "PageFieldDisplayOrder": 1,
```

Comment:
Let's have this above applicant and we need to sort out the PageFieldDisplayOrder to be numbered 1, 2, 3...

Reply:
👍